charts/s1-agent/values.yaml

```diff
@@ -65,6 +65,13 @@ agentInjection:
     namespaceSelector:
       matchLabels:
         agent-injection-enabled: "true"
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 100m
+      memory: 100Mi
```

Comment:
The resources for the agent container will be determined in another way (we'll take it offline)
Also, I think we can use the already present "resources" section in the Value.yaml field agent.resources